src/main/java/capstone/mfslbackend/service/LoginService.java

```diff
@@ -0,0 +1,71 @@
+package capstone.mfslbackend.service;
+
+import capstone.mfslbackend.DTO.UserDTO;
+import capstone.mfslbackend.model.User;
+import capstone.mfslbackend.repository.UserRepository;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.stream.Collectors;
+
+@Service
+@Transactional
+@AllArgsConstructor
+@Slf4j
+public class LoginService {
+
+    private final AuthenticationManager authenticationManager;
+
+    private final AccessTokenService accessTokenService;
+
+    private final RefreshTokenService refreshTokenService;
+
+    private final UserRepository userRepository;
+
+    @Transactional
```

Comment:
@Transactional on the class means you don't need it because it automatically applies to all public methods